src/app/pages/SearchPage/DataSearchResults.tsx

```diff
@@ -35,7 +51,7 @@ const DataSearchResults: React.FC<DataSearchResultsProps> = ({
   const Card = compact ? DataCard : DataList
 
   // Get the total count for all data types
-  const totalCount = results.length ? [0, ...results].reduce((acc, cur) => acc + cur.count) : 0
+  const totalCount = results.length ? results.reduce((acc, { count }) => acc + count, 0) : 0
```

Comment:
Nice cleanup! I think it could actually be reduced even more by removing the ternary entirely since a `reduce` on an empty array is a no-op. 

```suggestion
  const totalCount = results.reduce((acc, { count }) => acc + count, 0) 
```

Reply:
Could do that, but it will break when `results` is anything but an array.

Reply:
In this case we're relying on TypeScript to guarantee that this value is never anything else than an array (see the `DataSearchResults` interface). So checking it again here at runtime is not needed, unless of course the response can contain other values. In that case the type should be updated to reflect that.